core/common/src/test/java/alluxio/heartbeat/CronTimerTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.heartbeat;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.mock;
+
+import alluxio.Constants;
+import alluxio.clock.ManualClock;
+import alluxio.time.Sleeper;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.Duration;
+import java.util.Date;
+
+/**
+ * Unit tests for {@link CronTimer}.
+ */
+public final class CronTimerTest {
```

Comment:
CronTimer has been removed. Now need to unit test CronExpressionIntervalSupplier.

Reply:
Sure, rename it.